site/en/docs/extensions/mv3/manifest/input_component/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manifest - input_component"
+date: 2022-10-28
+updated: 
+description: Reference documentation for the input_component property of manifest.json.
+---
+
+An optional Manifest key enabling the use of the  [input.ime API](/docs/extensions/reference/input_ime/) (Input Method Editor) for use with ChromeOS. This allows your extension to handle keystrokes, set the composition, and open assistive windows. Including the key in the extension manifest is done under Permissions.
```

Comment:
> Including the key in the extension manifest is done under Permissions.

This seems incorrect. As I read it, this is saying that `"input_component"` is a top property inside an extension's manifest.json file, but that key is also a string that should be declared in the `"permissions"` array in the manifest. 

According to the [`input.ime` API docs](https://deploy-preview-3837--developer-chrome-com.netlify.app/docs/extensions/reference/input_ime/), developers must declare the `"input"` permission in the permission list. Perhaps this this should be reframed like this?

```suggestion
An optional Manifest key enabling the use of the  [`input.ime` API](/docs/extensions/reference/input_ime/) (Input Method Editor) for use with ChromeOS. This allows your extension to handle keystrokes, set the composition, and open assistive windows. Developers must also declare the `"input"` permission in the extension's `"permissions"` array. 
```